antsibull/build_changelog.py

```diff
@@ -527,6 +527,9 @@ def _get_porting_guide_bytes(changelog: Changelog) -> bytes:
                 " fully-qualified collection name (FQCN) in your playbooks as the explicit and"
                 " authoritative indicator of which collection to use as some collections may"
                 " contain duplicate module names."
+                " You can search the `index of all modules"
+                " <https://docs.ansible.com/ansible/latest/collections/index_module.html>`_"
```

Comment:
Since very soon (hopefully ;) ) `latest` will no longer be 2.10, but 3:
```suggestion
                " <https://docs.ansible.com/ansible/2.10/collections/index_module.html>`_"
```

Reply:
Good point.